backends/advanced/src/advanced_omi_backend/workers/base_audio_worker.py

```
@@ -0,0 +1,156 @@
+
+            # Trigger stop on consumer
+            self.logger.info("Stopping consumer...")
+            await self.consumer.stop()
+            
+            # Ensure consumer task finishes if it was running
+            if consumer_task in pending:
+                try:
+                    await consumer_task
+                except asyncio.CancelledError:
+                    pass
+                except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)